spec/api/comment_thread_spec.rb

```diff
@@ -592,7 +592,24 @@ def thread_result(id, params)
         comment.reload
         comment.endorsed.should == false
         comment.endorsement.should == nil
-        check_thread_result_json(nil, changed_thread, parse(last_response.body))
+        json_response = parse(last_response.body)
+        check_thread_result_json(nil, changed_thread, json_response)
```

Comment:
@robrap 
In that case I am wondering why `user.nil` case has been put there on first place (https://github.com/edx/cs_comments_service/blob/master/spec/spec_helper.rb#L258)
May be to improve performance; as according to API if user is not provided thread cannot be marked as read. Same goes with the test. As if thread is not marked as read then instead of making an extra query to get user `read_states` and checking its properties, it explicitly asserts `read` as `false` with user=nil.
What do you think?